WebSocketDemo-Shared/Server.swift

```diff
@@ -183,6 +184,12 @@ class Server: NSObject, ObservableObject, AVCaptureVideoDataOutputSampleBufferDe
       schemaName: Foxglove_CompressedImage.protoMessageName,
       schema: try! Data(contentsOf: Bundle(for: Self.self).url(forResource: "CompressedImage", withExtension: "bin")!).base64EncodedString()
     )
+    calibrationChannel = server.addChannel(
+      topic: "calibration",
```

Comment:
should we make this follow the same pattern to be autoselected in studio? (maybe it does already?)

Reply:
Skipping for now because we'd need two separate calibration topics.